modules/apps/site-initializer/site-initializer-raylife-ap/site-initializer-raylife-ap-test/src/testFunctional/tests/RaylifeAPApplicationsStatus.testcase

```diff
@@ -48,6 +46,105 @@ definition {
 		}
 	}
 
+	@description = "Story ID: LPS-159688 - Verify if the details of the Summary on the Bound application view appears."
+	@priority = 4
+	test AssertCanOpenBoundAppDetails {
```

Comment:
Here is just a suggestion, maybe the name of the test could be: AssertThatTheUserCanOpenLinkedApplicationDetails